src/dpp/utility.cpp

```diff
@@ -550,6 +550,22 @@ namespace dpp {
 		    return "<@&" + std::to_string(id) + ">";
 		}
 
+		std::string message_url(const snowflake& guild_id, const snowflake& channel_id, const snowflake& message_id){
+			return url_host + "/channels/" + std::to_string(guild_id) + "/" + std::to_string(channel_id) + "/" + std::to_string(message_id);
```

Comment:
You should add a check here to return an empty string if one of the ids are 0, otherwise it might return an invalid url

Reply:
Yeah I agree with this, I'd do it for all of them too :)

Reply:
Should it be for all methods, or only for message? (i guess for all, just to make sure)

Reply:
> Should it be for all methods, or only for message? (i guess for all, just to make sure)

Yeah, all would be ideal :)